### ISSUE TITLE
details/modified must be datetime, not str 

### ISSUE THREAD

**Issue by zopyx** [PyFilesystem/webdavfs#8]
`details/modified` is supposed to be a datetime instance:

https://docs.pyfilesystem.org/en/latest/info.html

In my case (ExistDB as backend) `modified` is a raw date string.

```
{'_to_datetime': <function epoch_to_datetime at 0x7f179d285230>, 'raw': {'access': {}, 'details': {'size': 311497, 'type': 2, 'modified': u'Sun, 12 Nov 2017 16:28:54 GMT', 'created': u'2017-11-12T16:28:54Z'}, 'basic': {'is_dir': False, 'name': u'html.zip'}}, 'namespaces': frozenset(['access', 'details', 'basic'])}
```

How can we better handle this? Trying to convert the `modified` string using and RFC 822 date parser and then using an ISO8601 parser to datetime?

**Comment by willmcgugan** [comment]
All times in the raw info should be specified as a float containing an epoch time. The Info class will convert those to datetime objects. There are functions in [time.py](https://github.com/PyFilesystem/pyfilesystem2/blob/master/fs/time.py) to convert between datetime and epoch. More information [here](https://docs.pyfilesystem.org/en/latest/info.html#raw-info)

**Comment by zopyx** [comment]
I fixed this in 0.3.2.
